include/oneapi/dpl/complex

```diff
@@ -21,7 +21,15 @@ namespace oneapi
 {
 namespace dpl
 {
+using ::std::abs;
+using ::std::arg;
 using ::std::complex;
+using ::std::conj;
+using ::std::exp;
+using ::std::imag;
+using ::std::real;
 } // namespace dpl
 } // namespace oneapi
+namespace dpl = oneapi::dpl;
```

Comment:
@akukanov Alexey, is it required to add this using here?

Reply:
Yes, it creates a short alias to the main oneDPL namespace and should exist in each public header.

Reply:
OK
Now this using absent in type_traits, async, cmath, cstddef, cstring, memory, optional, ranges, ratio, tuple, utility and array files.
Should we add this using into these files too ?

Reply:
Done